config/initializers/recaptcha.rb

```diff
@@ -1,9 +0,0 @@
-# config/initializers/recaptcha.rb
```

Comment:
Also beware here, looks like you've removed some config files we need. This may have happened during a merge from upstream as Ananyo's mentioned; rebasing can give you a cleaner set of commits -- this page has some helpful workflows for rewinding and syncing master before rebasing your work: https://publiclab.org/wiki/contributing-to-public-lab-software